plugins/outputs/mqtt/mqtt.go

```diff
@@ -225,7 +225,7 @@ func (m *MQTT) collectField(hostname string, metrics []telegraf.Metric) []messag
 	for _, metric := range metrics {
 		topic, err := m.generator.Generate(hostname, metric)
 		if err != nil {
-			m.Log.Warnf("Generating topic name failed: %w", err)
+			m.Log.Warnf("Generating topic name failed: %v", err)
```

Comment:
Start log message with capital letter? (for line below)

Reply:
???